_posts/2024-02-01-sswg-update-2024.md

```diff
@@ -0,0 +1,71 @@
+
+We've seen a lot of progress on the adoption of Structured Concurrency. Most new libraries are adopting it, and we're working on a guide to help existing libraries adopt it as well.
+
+Rewrites of ServiceLifecycle, Hummingbird and GRPC are under way that are implemented entirely with Structured Concurrency. Moving to structured concurrency allows NIO to become more of an implementation detail. In addition, structured concurrency provides many advantages to libraries and the adopters, which will be further elaborated on in our guides.
+
+## Tooling Improvements
+
+Tooling has seen some tremendous improvements in 2023. We've seen the introduction of Swiftly, Dependabot for Swift and Chiseled Containers.
+
+In addition, with the introduction of Package Plugins, we've seen the OpenAPI Generator make an entry. This plugin allows you to generate Swift boilerplate code from an OpenAPI specification. This is a great way for building both HTTP clients and servers.
+  
+## Survey
+
+Last year we've held a survey to get a better understanding of the community's needs and ideas. We're currently working on sharing some of our findings, and will be using the results to guide our work in 2024.
+
+# Ecosystem
+
+- Incubated Packages
+
+# Goals for 2024
+
+## Adoption of Structured Concurrency
+
+Adoption of Structured Concurrency has been going strong. In the coming year, we'll be focusing on adopting structured concurrency in more libraries across the ecosystem. In the end goal, SwiftNIO is an implementation detail to the extend possible.
+
+We're still lacking some guides. In order to drive further adoption, we'll be focusing on sharing our knowledge and expertise needed to adopt Structured Concurrency.
+
+## Preparing for Swift 6
+
+In preparation for Swift 6, we'll be looking at the impact of the new concurrency features on the ecosystem. By enabling strict concurrency checking, and working these warnings to zero, we'll be able to prepare the ecosystem for Swift 6.
+
+## Standardised HTTP Server and Middlewares
+
+In 2023, we've extensively prepared work on a standardised HTTP Server and Middlewares. Since then, Hummingbird 2 has been working on a rewrite that is entirely based on Structured Concurrency. The experiences from that rewrite will be used to further improve the standardised HTTP Server and Middlewares designs.
+
+## Marketing Swift on the Server
+
+Marketing Swift on the Server is a big focus area for 2024. With the introduction of the redesigned navigation of the [swift.org](swift.org) website, we'll be able to better showcase Swift on the Server. 
+
+We're looking to create more content for the ecosystem, such as integrated example projects and showcases of Swift-Server projects.
+
+In addition, we're looking to connect developers by organising online user group meetings. We hope to share more about that soon.
+
+Finally, we're expecting a new edition of the [Swift Server Conference](https://serversideswift.info) in 2024.
```

Comment:
Mention "in europe" and ask the question if we could organize something like this in the US? (call to arms kind of)